src/test/java/core/basesyntax/services/impl/CsvWriterImplTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.services.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CsvWriterImplTest {
+    private static final String VALID_FILE_NAME = "src/test/resources/testWrite.csv";
+    private static final String INVALID_FILE_NAME = "";
+    private static final String REPORT = "Name, Age, Email\nJohn, 30, john@example.com";
+    private CsvWriterImpl csvWriter;
+
+    @BeforeEach
+    void setUp() {
+        csvWriter = new CsvWriterImpl();
+    }
+
+    @Test
+    void testWrite_Ok() throws IOException {
```

Comment:
Here is the same comment regarding naming - please check all other test names and amend it